### PR TITLE
Bump golangci-lint, fix discovered problems

### PR DESCRIPTION
**Pull request by paskal**
Fixes red master.

### CODE REVIEW

File: libstring/libstring_test.go [hunk 1]
```diff
@@ -169,6 +169,7 @@ func TestCanonicalizeIP(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
+		tt := tt
```
Comment (didip) [inline libstring/libstring_test.go:172]: Question, why is this needed?

Comment (paskal) [reply libstring/libstring_test.go:172]: [Here](https://eli.thegreenplace.net/2019/go-internals-capturing-loop-variables-in-closures/) and [here](https://stackoverflow.com/a/68559720/961092) is some information about issue golangci-lint reported.
